packages/react-components/lib/themes/global-theme.tsx

```diff
@@ -2,22 +2,24 @@ import { withStyles } from '@material-ui/core/styles';
 
 export const GlobalCss = withStyles((theme) => ({
   '@global': {
+    // override accordian background to distinguish it from onipanel
```

Comment:
Sounds like we should change the palette of omnipanel or the accordions used in it instead. The global css changes every instance of the component, it should not depend specific components.